src/_macosx.m

```diff
@@ -991,74 +939,35 @@ -(void)zoom:(id)sender
 }
 
 -(void)configure_subplots:(id)sender
-{   PyObject* canvas;
-    View* view;
-    PyObject* size;
-    NSRect rect;
+{
+    View* master_view, * canvas_view;
+    PyObject* master = NULL, * canvas = NULL, * size = NULL;
```

Comment:
Don't really like mixing pointers in one line like this, mostly because we use `type* name` in this file, and that really obscures pointers as opposed to `type *name`.

Reply:
sure